src/graph/Graph.tsx

```diff
@@ -83,20 +75,23 @@ const Graph2 = ({ data, config /* , chromaticScale */ }: Props) => {
     setGraph(highlightGraph(highlightEdges))
   }
 
-  const onPaneClick = () => setGraph(resetHighlight)
+  const onPaneClick = () => {
+    setGraph(resetHighlight)
+    setDrawerVisibility(false)
+  }
 
-  const onSearchNode = () => {
-    if (!searchValue) {
+  const onSearchNode = ({ val, err, onError }: SearchArgs) => {
+    if (!val) {
       return
     }
 
-    const node = graph.nodes.find((node: Node) => node.data?.label.indexOf(searchValue) !== -1)
+    const node = graph.nodes.find((node: Node) => node.data?.label.indexOf(val) !== -1)
```

Comment:
What if we want to found a group of nodes that have the same subname. I suggest using a filter instead find

Reply:
applied some change, have a look